src/edge/tests/test_importer.py

```diff
@@ -248,50 +248,77 @@ def test_import_feature_ending_at_last_base(self):
 
 class QualifierTest(TestCase):
 
-    def import_with_qualifiers(self, qualifiers):
+    def import_with_qualifiers(self, qualifiers, phase="."):
         data = """##gff-version 3
-chrI\tTest\tcds\t30\t80\t.\t-\t.\t%s
+chrI\tTest\tcds\t30\t80\t.\t-\t%s\t%s
```

Comment:
I forgot what does the %s mean here?

Reply:
I think %s is replaced by values in the python code, so within test we can be consistent.

Reply:
gotcha!